conf/config.level0.neon

```diff
@@ -21,6 +21,7 @@ rules:
 	- PHPStan\Rules\Methods\ExistingClassesInTypehintsRule
 	- PHPStan\Rules\Properties\AccessStaticPropertiesRule
 	- PHPStan\Rules\Variables\ThisVariableRule
+	- PHPStan\Rules\Statements\EchoRule
```

Comment:
Hi! This is a great first draft and the general approach is correct, this is how I planned to implement it :)

The rule should be on level 2 to be consistent with other similar rules.